src/base/http/requestparser.cpp

```diff
@@ -53,6 +53,11 @@ namespace
 {
     const QByteArray EOH = CRLF.repeated(2);
 
+    bool isDigit(const char c)
+    {
+        return std::isdigit(c) != 0;
```

Comment:
In fact, I'm surprised that anyone still needs to use functions with such signatures nowadays. Aren't there any modern (C++ style) predicates for this purpose?